CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Remove LegacyESVersion.V_7_10_ Constants ([#5018](https://github.com/opensearch-project/OpenSearch/pull/5018))
 - Remove Version.V_1_ Constants ([#5021](https://github.com/opensearch-project/OpenSearch/pull/5021))
 - Remove custom Map, List and Set collection classes ([#6871](https://github.com/opensearch-project/OpenSearch/pull/6871))
+- Remove outdated DEFAULT_MAPPING_ERROR_MESSAGE error message ([#7008](https://github.com/opensearch-project/OpenSearch/pull/7008))
```

Comment:
Can probably skip the changelog since this is code cleanup

Reply:
@andrross Can I remove it from `CHANGELOG.md` file now? (I guess the GH label does the magic now, correct?)

Reply:
Yup, the label just tells the github action not to fail without a changelog entry.

FYI, the [CONTRIBUTING guide](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#changelog) has been updated with more details about this process.

Reply:
Removed `CHANGELOG.md` changes.